x/pool-incentives/client/proposal_handler.go

```diff
@@ -0,0 +1,11 @@
+package client
+
+import (
+	"github.com/c-osmosis/osmosis/x/pool-incentives/client/cli"
+	"github.com/c-osmosis/osmosis/x/pool-incentives/client/rest"
+	govclient "github.com/cosmos/cosmos-sdk/x/gov/client"
+)
+
+var AddPoolIncentivesHandler = govclient.NewProposalHandler(cli.NewCmdSubmitAddPoolIncentivesProposal, rest.ProposalAddPoolIncentivesRESTHandler)
```

Comment:
Would be better to clearly name it as "AddPoolIncentivesProposalHandler" so that name describes it as a proposal handler.